substrate/frame/proxy/src/benchmarking.rs

```diff
@@ -81,11 +89,13 @@ fn add_announcements<T: Config>(
 }
 
 benchmarks! {
+	where_clause { where T: Config + pallet_balances::Config }
```

Comment:
There are different cases to handle this, but specifically requiring that pallet balances is deployed does not seem very clean to me.  
You can either create a [`BenchmarkHelper`](https://github.com/paritytech/polkadot-sdk/blob/954150f3b5fdb7d07d1ed01b138e2025245bb227/bridges/snowbridge/pallets/inbound-queue/src/lib.rs#L130) trait with a `Currency` or `Fungible` config item and then have that feature-gated in the main config, or require it here.

Reply:
PS what Muharem wrote about `Consideration::ensure_successful` is better and exactly what it is intended for.